src/fides/api/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -260,21 +262,14 @@ def queue_privacy_request_to_propagate_consent(
     executable_data_uses = [
         ec.data_use for ec in executable_consents or [] if ec.executable
     ]
-    executable_consent_preferences: List[Dict] = [
-        pref.dict()
-        for pref in consent_preferences.consent or []
-        if pref.data_use in executable_data_uses
-    ]
 
-    if not executable_consent_preferences:
-        logger.info(
-            "Skipping propagating consent preferences to third-party services as "
-            "specified consent preferences: {} are not executable.",
-            [pref.data_use for pref in consent_preferences.consent or []],
-        )
-        return None
+    executable_consent_preferences: List[Dict] = []
+    for pref in consent_preferences.consent or []:
+        if (
+            pref.data_use and pref.data_use in executable_data_uses
+        ) or pref.privacy_notice_id:
+            executable_consent_preferences.append(pref.dict())
```

Comment:
This is new, both for the old workflow (save w.r.t. data use) and save w.r.t privacy notice, I am creating a privacy request regardless for record-keeping instead of just when there are consent preferences to propagate. If there are no consent preferences on the request, the privacy request will quickly exit in a "complete" state.